tests/kani/SizeAndAlignOfDst/main_assert_fixme.rs

```diff
@@ -1,12 +1,13 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 // This test takes too long with all the std symbols. Use --legacy-linker for now.
```

Comment:
This should be updated. Maybe you can remove and explain that `original` takes too long.

Reply:
Will do.